x-pack/plugins/threat_intelligence/cypress/tasks/es_archiver.ts

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
took it from `security_solution` with the exception of one method which I wasn't sure the usecase of